
<launch>

  <arg name="resolution"   default=".2"/>

	<node pkg="octomap_server" type="octomap_server_node" name="octomap_server">

		<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
		<param name="frame_id" type="string" value="map" />

		<param name="resolution"               value="$(arg resolution)" />
		<param name="sensor_model/max_range"   value="5.0" />
    <param name="sensor_model/min_range"   value="0.1" />
    <param name="sensor_model/hit"         value=".8"/>
    <param name="sensor_model/miss"        value=".49"/>
    <param name="sensor_model/min"         value=".01"/>
    <param name="sensor_model/max"         value=".99"/>
    <param name="filter_speckles"          value="true"/>
    <param name="filter_ground"            value="false"/>

		<!-- data source to integrate (PointCloud2) -->
		<remap from="cloud_in" to="octomap_in" />

	</node>
</launch>
